gapis/trace/android/adreno/profiling_data.go

```diff
@@ -54,6 +54,29 @@ func ProcessProfilingData(ctx context.Context, processor *perfetto.Processor, ca
 	return &service.ProfilingData{Slices: slices, Counters: counters}, nil
 }
 
+func extractTraceHandles(ctx context.Context, replayHandles *[]int64, replayHandleType string, handleMapping *map[uint64][]service.VulkanHandleMappingItem) {
```

Comment:
Not new in this change, but can we eventually be consistent about signedness of handle types?

Reply:
I completely agree with you on this one but we have an issue there.

In all AGI, Vulkan Handles are treated as unsigned but in perfetto.proto, we don't have unsigned values. I think some preliminary work is required in perfetto side to send values unsigned.

I am happy to change this in another pr after that work has been done.